src/constant/index.js

```diff
@@ -207,7 +207,7 @@ const SCOPES = [
   },
   {
     id: "customers:own",
-    description: "Permission for managing customers identities",
+    description: "Permission for owning and managing customer identities. It allows for creating new customers via the Customer Chat API.",
```

Comment:
@bkubiak

Reply:
I have some doubts regarding "via the Customer Chat API." - in customer-api there's no method for creating a customer explicitly. You just use their identity (impersonate them) and login, start_chat, etc (this would automatically create a new customer if they didn't exist).

The part regarding customer creation an equivalent of customers:rw in agent-api.